app/models/chargeback_container_project.rb

```diff
@@ -114,4 +110,8 @@ def self.report_col_options
       "total_cost"            => {:grouping => [:total]}
     }
   end
+
+  def tags
```

Comment:
> @zeari where/how does this method get used?

The report generator uses this method to fill in the tag columns here: 
https://github.com/ManageIQ/manageiq/blob/master/app/models/miq_report/generator.rb#L697